src/utils/analysis-utils.ts

```diff
@@ -434,8 +436,25 @@ export async function loadFeaturesClientSide(
 }
 
 export function getAnalysisTableColumns(
-  analysisResult: BaselineLayerResult,
+  analysisResult: BaselineLayerResult | PolygonAnalysisResult,
 ): Column[] {
+  if ('tableColumns' in analysisResult) {
+    return analysisResult.tableColumns.map(column => {
+      return {
+        id: column as any,
```

Comment:
avoid any. Can't we fix up the underlying type to type columns properly?

Reply:
fixed
<img width="551" alt="Screen Shot 2022-04-03 at 11 32 26 AM" src="https://user-images.githubusercontent.com/4313463/161435566-7b1ee1ee-4795-4ed1-a4b2-afe7c65209eb.png">